docker/app/local.Dockerfile

```diff
@@ -1,27 +1,25 @@
-FROM bitnami/python:3.7-prod
+FROM python:3.7-slim
 
+# Allow service to handle stops gracefully
 STOPSIGNAL SIGQUIT
 
+# Set pip to have cleaner logs and no saved cache
 ENV PIP_NO_CACHE_DIR=false \
     PIPENV_HIDE_EMOJIS=1 \
     PIPENV_NOSPIN=1
 
-# Create a user.
+# Create non-root user
 RUN useradd --system --shell /bin/false --uid 1500 pysite
 
-# Install prerequisites needed to complete the dependency installation.
-RUN install_packages git gcc libc-dev libpq-dev
-
-# Copy the project files into the working directory.
+# Copy the project files into working directory
 WORKDIR /app
 COPY . .
 
-# Update setuptools by removing egg first, add other dependencies
-RUN rm -r /opt/bitnami/python/lib/python3.*/site-packages/setuptools* && \
-    pip install -U setuptools
-RUN pip install pipenv uwsgi
-RUN pipenv install --system --deploy
+# Install project dependencies
+RUN pip install -U pipenv pyuwsgi && pipenv install --system --deploy
```

Comment:
Would moving `RUN pip install -U pipenv pyuwsgi` above `COPY` help with layer caching?

Reply:
Yeah likely. Good idea.

Reply:
This is now actioned.